package/README.md

```diff
@@ -224,12 +330,14 @@ postgres-searchbox installed. See the 3 files:
 
 - `examples/with-nextjs/pages/api/search.ts`
 - `examples/with-nextjs/pages/movies.tsx`
-- `examples/with-nextjs/styles/Movies.module.css`
+- `examples/with-nextjs/pages/store.tsx`
 
 In `examples/with-nextjs` you can `yarn && yarn dev` to get the dev. server running.
-You can see the movies page at http://locaalhost:3000/movies
 
-NextJS can import the `package/build/*.js` files, to keep them up to date run `yarn build:watch-swc` from a 2nd terminal.
+- You can see the movies page at http://locaalhost:3000/movies
+- You can see the store page at http://locaalhost:3000/store
+
+NextJS can import the `package/build/*.js` files, to keep them up to date run `yarn dev` from a 2nd terminal.
```

Comment:
This is related to https://github.com/dekimir/postgres-searchbox/pull/12#discussion_r1180483442 . I don't think I know the best way to do this.

At the momement the yarn dev is needed in the package dir because nextjs is importing from the build directory. Without it, the build directory files don't get updated. So nextjs doesn't know when to rebuild.

Reply:
Ah, that makes sense. Then this is fine. I'll rephrase it later to make it a bit clearer, but what it says is true.